extensions/git/src/commands.ts

```diff
@@ -3351,9 +3351,17 @@ export class CommandCenter {
 		}
 	}
 
-	@command('git.createWorktree', { repository: true })
-	async createWorktree(repository: Repository): Promise<void> {
-		await this._createWorktree(repository, undefined, undefined);
+	@command('git.createWorktree')
+	async createWorktree(): Promise<void> {
+		const mainRepository = this.model.repositories.find(repo =>
```

Comment:
There is a scenario in which you might have multiple "main" repositories opened in a workspace and if that is the case here you would always pick the first one. Let's bring back the picker for the time being until we come up with a better solution.